src/benchmarks.rs

```diff
@@ -0,0 +1,408 @@
+			.dispatch_bypass_filter(RawOrigin::None.into())
+			.expect("dispatch succeeded");
+
+		RelayPallet::<T>::on_finalize(4u32.into());
+
+		// The new user
+		let new_user = create_funded_user::<T>("user", SEED+1, 0u32.into());
+
+	}:  _(RawOrigin::Signed(caller.clone()), new_user.clone())
+	verify {
+		assert_eq!(Pallet::<T>::accounts_payable(&new_user).unwrap().total_reward, (100u32.into()));
+	}
+
+	associate_native_identity {
+		let batch = max_batch_contributors::<T>();
+		let x in 2..MAX_USERS;
+		// Fund pallet account
+		let total_pot = 100u32*x;
+		fund_specific_account::<T>(Pallet::<T>::account_id(), total_pot.into());
+
+		// Create x contributors
+		let contributors = create_contributors::<T>(x-1);
+
+		// Insert them
+		insert_contributors::<T>(contributors)?;
+
+		// The caller that will associate the account
+		let caller: T::AccountId = create_funded_user::<T>("user", MAX_USERS-x-1, 100u32.into());
+
+		// Create a fake sig for such an account
+		let (relay_account, signature) = crate_fake_sig::<T>(caller.clone());
+
+		// Push this new contributor
+		let mut new_cont = Vec::new();
+		new_cont.push((relay_account.clone().into(), None, 100u32.into()));
+		insert_contributors::<T>(new_cont)?;
+
+		// Clonse initialization
+		close_initialization::<T>(10u32.into())?;
+
+		// First inherent
+		let first_block_inherent = create_inherent_data::<T>(1u32);
+		RelayPallet::<T>::on_initialize(T::BlockNumber::one());
+		RelayPallet::<T>::create_inherent(&first_block_inherent)
+			.expect("got an inherent")
+			.dispatch_bypass_filter(RawOrigin::None.into())
+			.expect("dispatch succeeded");
+		RelayPallet::<T>::on_finalize(T::BlockNumber::one());
+		Pallet::<T>::on_finalize(T::BlockNumber::one());
+
+	}:  _(RawOrigin::Signed(caller.clone()), caller.clone(), relay_account.into(), signature)
+	verify {
+		assert_eq!(Pallet::<T>::accounts_payable(&caller).unwrap().total_reward, (100u32.into()));
+	}
+
+}
+#[cfg(test)]
+mod tests {
+	use super::*;
+	use crate::mock::Test;
+	use frame_support::assert_ok;
+	use sp_io::TestExternalities;
+
+	pub fn new_test_ext() -> TestExternalities {
+		let t = frame_system::GenesisConfig::default()
+			.build_storage::<Test>()
+			.unwrap();
+		TestExternalities::new(t)
+	}
+
+	#[test]
+	fn bench_init_reward_vec() {
+		new_test_ext().execute_with(|| {
+			assert_ok!(test_benchmark_initialize_reward_vec::<Test>());
+		});
+	}
+	#[test]
+	fn complete_initialization() {
+		new_test_ext().execute_with(|| {
+			assert_ok!(test_benchmark_complete_initialization::<Test>());
+		});
+	}
+	#[test]
+	fn bench_claim() {
+		new_test_ext().execute_with(|| {
+			assert_ok!(test_benchmark_claim::<Test>());
+		});
+	}
+	#[test]
+	fn update_reward_address() {
+		new_test_ext().execute_with(|| {
+			assert_ok!(test_benchmark_update_reward_address::<Test>());
+		});
+	}
+	#[test]
+	fn associate_native_identity() {
+		new_test_ext().execute_with(|| {
+			assert_ok!(test_benchmark_associate_native_identity::<Test>());
+		});
```

Comment:
some of these are named `bench_$CALL_NAME` and some of them are just `$CALL_NAME`, could you make the naming conventions consistent